share/metainfo/net.lutris.Lutris.metainfo.xml

```diff
@@ -5,7 +5,6 @@
   <project_license>GPL-3.0-or-later</project_license>
   <metadata_license>CC0-1.0</metadata_license>
   <translation type="gettext">lutris</translation>
-  <developer_name translatable="no">Lutris Team</developer_name>
```

Comment:
Please do not remove this. 

I assume this is because of a lint in appstreamcli, it is only a warning.